Fix RPC behavior in multiplayer component tutorial (#1450)
o3de/o3de.org: content/docs/learning-guide/tutorials/multiplayer/first-multiplayer-component.md

    ```c++
    void MyFirstNetworkComponent::OnUpTimeChanged(double uptime)
    {
        AZ_Printf("MyFirstNetworkComponent", "client = %f\n", uptime);
        static_cast<MyFirstNetworkComponentController*>(GetController())->SendConfirmUptime(uptime);
    }
    ```

> 1. Finally, add a check to `MyFirstNetworkComponentController::OnActivate` to ensure it is only connecting to the TickBus while it's executing on the server (e.g., is not the autonomous player client, which will also have a controller):

Reviewer: ```suggestion
1. Finally, add a check to `MyFirstNetworkComponentController::OnActivate` to ensure `MyFirstNetworkComponentController` is only connecting to the `TickBus` while it's executing on the server. For example, it's connecting to the server rather than the autonomous player client, which also has a controller):
```
- Can you verify that "it" refers to `MyFirstNetworkComponentController`?
- I expanded your example at the end, since we try to avoid using Latin abbreviations such as "e.g.". Want to make sure with you that the intent of the example there hasn't changed.

    ```c++
    void MyFirstNetworkComponentController::OnActivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
    {

>         if (!IsAutonomous()) // add me

Reviewer 2: did you want to leave the _//add me_ comment?

Reviewer 3: What is the value of this comment?

Author: that's the check that we're asking them to add at this point - the rest of the code in this block should already exist. But if we think that's not helpful or needs to be worded better I'm all for changing it

    </RemoteProcedure>
    </Component>
    ```

10. At this point you should be able to see the client and server logs being emitted in the editor console when running the game in Play Mode:

>     ![Console logs emitted while running game from editor with RPC](/images/user-guide/multiplayer/add_myfirstnetworkcomponent_run_game_with_rpc.png)

Reviewer 2: the doc's team might not like the hand drawn red arrow, they could probably clean that up

Author: ah, thanks for the heads up. I can remove that or make it look more polished.

Reviewer: I recommend ShareX as a screenshot tool that includes features like drawing arrows. Here's some additional guidelines on [submitting media with image annotations](https://www.o3de.org/docs/contributing/to-docs/style-guide/media/#image-annotations).

Reviewer 4: Its not 100% clear what the red arrow is pointing it at in the screenshot, it appears to be pointing at a specific line, where as I think you are trying to highlight the editor logs box

Reviewer 3: Wouldn't be easy to just include the text of the log instead of a picture of a log?

Author: > Wouldn't be easier to just include the text of the log instead of a picture of a log?

I like the picture b/c it shows you where in the editor this would show up. Someone less familiar with the editor layout or running a game from it may not have a concrete idea for where to look.

Reviewer 5: Maybe draw a red box around the log lines we want to highlight instead of using the red arrow? As Pip pointed out, the red arrow might be a little confusing since it seems pointing to a specific line.
